src/pages/styling.mdx

```diff
@@ -48,10 +48,11 @@ If your app is small enough, it may be simpler to reach for a headless solution
 
 ## Headless Component Libraries
 
+- [Base UI](https://www.npmjs.com/package/@mui/base)
```

Comment:
I re-ordered these to be in order of popularity on npm weekly downloads. Seems to make the most sense to me to do it that way.

Reply:
A good chunk of the downloads of Base UI is coming indirectly from Material UI. So, it's not accurate. Alphabetically, it could be more accurate.